content/control-api/index.textile

```diff
@@ -0,0 +1,229 @@
+---
+title: Control API
+meta_description: "The Control API is a REST API that enables you to manage your Ably account programmatically. This is the Control API Reference guide."
+meta_keywords: "Ably, Control API, account management, control, REST API, OpenAPI Specification, OAS3."
+section: control-api
+index: 0
+jump_to:
+  Help with:
+    - OpenAPI Specification#oas3
+    - Authentication#authentication
+    - Account and application IDs#ids
+    - Obtain token details using the API#obtain-token
+    - Quickstart#quickstart
+    - Rate limits#rate-limits
+    - See also#see-also
+---
+
+The Control API is a REST API that enables you to manage your Ably account programmatically. 
+
+Using the Control API you can manage:
+
+* Your applications
+* Keys for an application
+* "Namespaces":/general/channel-rules-namespaces (for channel rules)
+* "Queues":/general/queues
+* "Reactor rules":/general/events
+
+Repetitive operations such as creating or updating apps, enumerating queues, and other tasks can be automated using the Control API.
+
+In order to use the Control API you need to first create an access token in the "Ably dashboard":https://ably.com/accounts/any. However, once you have done this you can use the Control API to manage your account without having to interact with the dashboard for many functions.
+
+h2(#oas3). OpenAPI Specification (OAS3)
+
+The OpenAPI Specification (OAS3) for this API can be found in the Ably OpenAPI Specifications "GitHub repository":https://github.com/ably/open-specs.
```

Comment:
I believe we should refer to this as the "OpenAPI document for this API", rather than the "OpenAPI Specification for this API". The spec is a standard, the document is the description of the API. See [terminology](https://swagger.io/specification/).

Reply:
TIL!